about/introduction.rst

```diff
@@ -80,7 +80,7 @@ unbalanced distribution of contents – but the way it is split up should be
 relatively intuitive:
 
 - The :ref:`sec-general` section contains this introduction as well as
-  information about the engine, its history, its licensing, authors, etc. It
+  the information about the engine, its history, its licensing, authors, etc. It
```

Comment:
I think the previous form was correct already. "information" is often used in a non-numerable form.

Reply:
I've specifically double-checked this before adjusting, and found out that if you refer to a specific information, which we do in the cases I've adjusted, you can use the article.